mailbox/plugin/quota-mailing/src/main/java/org/apache/james/eventsourcing/AggregateId.java

```diff
@@ -20,4 +20,5 @@
 package org.apache.james.eventsourcing;
 
 public interface AggregateId {
+    String asAggregationKey();
```

Comment:
It's AggregateKey, not AggregationKey, it's important because it refers to EventSourcing official wording

Reply:
Sorry I'm not familiar with such wording but of course, you're right.